src/domain/ledger/index.types.d.ts

```diff
@@ -44,7 +44,6 @@ type LedgerTransaction = {
   readonly feeUsd: number
 
   // for IntraLedger
-  readonly walletPublicId?: WalletPublicId
```

Comment:
This was originally meant to be the recipient's `walletPublicId`. Should we instead rename this to something like `payeeWalletId` or `recipientWalletId`?